project.md

```diff
@@ -7,62 +7,195 @@ good way to meet their needs.
-of language design)?_
+I think a significant portion of the project will be based on the "language"
+aspects since a key part of this is creating features which will essentially be
+language components that make different designs easier.
+I will spend much of my time debating which features would be most useful to a
+user and how I can make the syntax clear, concise, and non-restrictive.
+There will, however, also definitely be "systems" aspect that I will need to
+consider.
+For each feature, I need to think about how I will actually render it in the web
+languages, but since I plan to make the syntax fairly similar to CSS/JS, I hope
+this part will not be too difficult or time consuming.
+Thus, I would guess 60% of the time would be spent on "language" and 40% on
+"systems".
 
 ### Scope
 
-_How big an idea is this? How ambitious is this project?_
+If I wanted to make a complete replacement for presentation software in this one
+class, I believe this would be highly ambitious, but since I just plan to start
+the process and make a few unique features, I think the scope is much more
+manageable.
+What's nice about this project is that the scope can be grown as desired simply
+by adding more features.
+I think the basic features I want to add may be a little ambitious, but should
+definitely be doable in the amount of time we have.
 
 ### Benefits and drawbacks
 
-_Why might this be a good idea for a project? Why might this not be a good idea
-project?_
+I think the largest benefit is that I am excited about this project, so I will
+be eager to spend time working on it and putting in lots of effort to see it
+grow.
+Additionally, it is helpful that I am pretty familiar with web languages which
+makes the interpretation aspects of this project more straightforward than if I
+were trying to create an output that I was not already familiar with.
+I really like that this project would result in something that I could regularly
+use in my day-to-day life and share with others that may have similar
+reservations against presentation software.
+Finally, I like that the scope is adjustable.
+It keeps the project from being overwhelming, but also offers many ways to
+challenge me.
+When programming, I like having relatively immediate feedback on the
+functionality of what I'm writing, so it helps that I can create a basic start
+that will allow parsing and interpretation before I create all the features.
+
+In terms of drawbacks, there are a few.
+Firstly, I plan to output files in multiple languages which can make the
+processing a bit complicated.
+Additionally, since I plan to make this an external DSL, I won't have the added
+benefits of existing ways to compile the project which may make the result less
+user-friendly.
+Finally, since there seems to be no similar software related to this domain, it
+could suggest that a) this domain is particularly difficult to work with and
```

Comment:
Is it only outputting html files? It sounds like you're creating a language that makes code for a website maybe?